src/azure-cli/setup.py

```diff
@@ -136,6 +137,7 @@
     'azure-synapse-managedprivateendpoints~=0.3.0',
     'azure-synapse-spark~=0.2.0',
     'chardet~=3.0.4',
+    'colorama~=0.4.4',
```

Comment:
`colorama` was removed from Knack for non-Windows systems (https://github.com/microsoft/knack/pull/249), but Azure CLI still needs it.